crates/wash-cli/src/dev.rs

```diff
@@ -2015,9 +2044,11 @@ pub async fn handle_command(
 
                     // Stop NATS
                     if let Some(mut nats) = nats_child {
+                        eprintln!("{} Stopping NATS...", emoji::HOURGLASS_DRAINING);
                         nats.kill().await?;
                     }
                 }
+                eprint!("✅ Dev session exited successfully");
```

Comment:
We don't want the newline here? But at lease we can use something from `emoji::`